tests/snapshots/files__loop_pass_through-optimize.snap

```diff
@@ -3,7 +3,7 @@ source: tests/files.rs
 expression: visualization.result
 ---
 @main(v0: int) {
-  v1_: int = const 1;
+  v1_: int = const 2;
```

Comment:
Why did this value change? It looks like we're comparing the loop variable against 10, but the rule looks for multiples of 4?

Reply:
Good catch, I'm not sure... I can't reproduce it locally

Reply:
I see now- it's equivalent because you can loop peel the loop once.

Reply:
But this rule shouldn't do peeling, right? It shouldn't match.

But if you can't repro locally maybe it was a different rule that caused it?